playbooks/templates/common/metadata_common.yaml

```diff
@@ -0,0 +1,10 @@
+metadata          :
+  RPC: {{ maas_rpc | default(true) }}
+  RPC_env_identifier: {{ maas_env_identifier | default('unkown')}}
+  RPC_env_identifier_alt: {{ maas_env_identifier_alt | default('unkown')}}
+  RPC_maas_version: {{  lookup('pipe', 'cd .. && git describe --tags --abbrev=0') }}
```

Comment:
This is a relative path, will this work in all use-cases?

Reply:
Based on my test it will read from the playbook directory so it will work.